libcaf_core/test/actor_lifetime.cpp

```diff
@@ -73,7 +73,8 @@ behavior tester(event_based_actor* self, const actor& aut) {
       // must be still alive at this point
       CAF_CHECK_EQUAL(s_testees.load(), 1);
       CAF_CHECK_EQUAL(msg.reason, exit_reason::user_shutdown);
-      CAF_CHECK_EQUAL(self->current_message().vals()->get_reference_count(), 1);
+      CAF_CHECK_EQUAL(self->current_message().vals()->get_reference_count(),
+                      static_cast<size_t>(1));
```

Comment:
In these cases, you could avoid the verbose cast by using `CAF_CHECK`. Or, my preferred solution, just use an unsigned literal and write `1u` instead of `static_cast<size_t>(1)`.